src/library_webgpu.js

```diff
@@ -559,7 +584,6 @@ var LibraryWebGPU = {
     var desc = {
       "label": undefined,
       "size": WebGPU.makeExtent3D(descriptor + {{{ C_STRUCTS.WGPUTextureDescriptor.size }}}),
-      "arrayLayerCount": {{{ gpu.makeGetU32('descriptor', C_STRUCTS.WGPUTextureDescriptor.arrayLayerCount) }}},
```

Comment:
This shouldn't be removed for now, it isn't part of the [PSA](https://hackmd.io/Et7xlhoaThmi8dEX_s-xSw?view) we sent, and it breaks the possibility to have applications that target both Dawn and emscripten at the same time. It was found by @hugoam.